targets/TARGET_RENESAS/TARGET_RZ_A1XX/TARGET_GR_LYCHEE/CMakeLists.txt

```
@@ -0,0 +1,52 @@
+# Copyright (c) 2020 ARM Limited. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+if(${MBED_TOOLCHAIN} STREQUAL "ARM")
+    set(LINKER_FILE device/TOOLCHAIN_ARM_STD/MBRZA1LU.sct)
+    set(STARTUP_FILE device/TOOLCHAIN_ARM_STD/startup_RZ_A1LU.S)
+    target_sources(mbed-core
+        INTERFACE
+            device/TOOLCHAIN_ARM_STD/weak_handler.S
+    )
```

Create a variable `WEAK_HANDLER_FILE` then add the variable to the main target_sources() just as done for `STARTUP_FILE`.
```suggestion
    set(WEAK_HANDLER_FILE device/TOOLCHAIN_ARM_STD/weak_handler.S)
```
The same change for GCC_ARM.